src/main/java/no/nav/familie/inntektsmelding/forespørsel/tjenester/task/SettForespørselTilUtgåttTask.java

```diff
@@ -52,7 +52,8 @@ public void doTask(ProsessTaskData prosessTaskData) {
             return;
         }
 
-        boolean skalOppdatereArbeidsgiverNotifikasjon =  forespørsel.getStatus() == ForespørselStatus.UNDER_BEHANDLING;
+        boolean skalOppdatereArbeidsgiverNotifikasjon = forespørsel.getStatus() == ForespørselStatus.UNDER_BEHANDLING;
         forespørselBehandlingTjeneste.settForespørselTilUtgått(forespørsel, skalOppdatereArbeidsgiverNotifikasjon);
+        MetrikkerTjeneste.loggForespørselLukkEkstern(forespørsel);
```

Comment:
Jeg er litt usikker på om vi ønsker enda en ny metrikk som teller dersom `skalOppdatereArbeidsgiverNotifikasjon=false` 🤔 Foreslår å begynne med dette

Reply:
tenker dette er fint for now